src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, Colorful {
+    protected String color;
+
+    public Figure() {
+        this.color = "white";
+    }
+
+    public interface Drawable {
+        void draw();
+    }
+
+    public interface Colorful {

```

Comment:
Why did you put this interface here? You've already created it in separate file.

Reply:
Not fixed